client/my-sites/stats/stats-notices/commercial-site-upgrade-notice.tsx

```diff
@@ -77,8 +63,8 @@ const CommercialSiteUpgradeNotice = ( { siteId, isOdysseyStats }: StatsNoticePro
 			<NoticeBanner
 				level="info"
 				title={ translate( 'Upgrade to Stats Commercial' ) }
-				onClose={ dismissNotice }
-				hideCloseButton
+				onClose={ () => {} }
+				hideCloseButton={ hideCloseButton }
```

Comment:
Would it make sense to just pass `false` here and get rid of `hideCloseButton` variable?

Reply:
sorry I meant true 😅

Reply:
> sorry I meant true 😅

haha yes, true indeed. I was happily going with false too 😆  good time to take a quick coffee break and reset my brain I think.